static/app/components/workflowEngine/form/control/index.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import {Fragment} from 'react';
+
+import {Flex} from 'sentry/components/container/flex';
+import Form from 'sentry/components/forms/form';
+import JSXNode from 'sentry/components/stories/jsxNode';
+import PriorityControl from 'sentry/components/workflowEngine/form/control/priorityControl';
+import {FormSection} from 'sentry/components/workflowEngine/form/section';
+import storyBook from 'sentry/stories/storyBook';
+import {space} from 'sentry/styles/space';
+
+export default storyBook('Form Controls', story => {
+  story('PriorityControl', () => (
+    <Fragment>
+      <p>
+        The <JSXNode name="PriorityControl" /> component allows users to configure issue
+        priority when a Monitor matches specific thresholds.
+      </p>
+
+      <Form hideFooter>
+        <Flex column gap={space(2)}>
+          <FormSection
+            title="Prioritize"
+            description="Update issue priority when the following thresholds are met:"
+          >
+            <PriorityControl name="priority" limit={3} />
+          </FormSection>
+
+          <FormSection
+            title="Prioritize"
+            description="Update issue priority when the following thresholds are met:"
+          >
+            <PriorityControl name="priority" limit={2} />
```

Comment:
It'd be cool if we could select the other limit's type.

For example with this configuration:
![Screenshot 2025-02-03 at 4 19 50 PM](https://github.com/user-attachments/assets/ded82c97-4524-4487-b80c-a37c4337e456)
it'd be cool if i could select `Med` instead of having it force to `High`, in the cases i never want to create a high priority issue. (I think issue priority is going to be how people do a low of workflow controls. by that i mean something like... "trigger pager duty for high priority issues only")

Reply:
This has been fixed after chatting through the interaction with Vu